src/components/confirm/confirm.stories.mdx

```diff
@@ -74,6 +74,171 @@ const MyComponent = ({ isOpen, handleClose, handleConfirm }) => (
   </Story>
 </Preview>
 
+
+### Single action
+
+<Preview>
+  <Story name="single action" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+
+## `destructive`
```

Comment:
Looks like you need to add `ml={2}` to this the sibling button in this example. 
<img width="416" alt="Screenshot 2021-02-08 at 12 10 29" src="https://user-images.githubusercontent.com/56251247/107217930-d45aca80-6a06-11eb-87ce-5147c923fc69.png">